src/Widgets/Helper/Factories/Settings/FileSettingFactory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+
+namespace Ceres\Widgets\Helper\Factories\Settings;
+
+
+class FileSettingFactory extends BaseSettingFactory
+{
+    public function __construct()
+    {
+        $this->withType('file');
+    }
+
+    /**
+     * @param boolean $showPreview
+     */
+    public function withPreview($showPreview)
```

Comment:
```suggestion
    public function withShowPreview($showPreview)
```
=> Einheitliche Benennung zur Doku